### ISSUE TITLE
Masthead a11y violations

### ISSUE THREAD

**Issue by hxltrhuxze** [Royal-Navy/design-system#1099]
- [ ] Ensures buttons have discernible text

**Comment by m7kvqbe1** [comment]
This issue doesn't appear to be being flagged anymore by AXE. Must have been resolved as part of earlier works.
